docker/test-images/zipkin-eureka/README.md

```diff
@@ -0,0 +1,12 @@
+## zipkin-eureka Docker image
```

Comment:
there is no docker image for eureka, and in fact as of 2.x they don't release a working war file anymore. At any rate, this server was easy to build, and now we can test without thinking too much.